fares-data-build-tool/create-data
# feat: CFD-305 Prices per sales offer package

# Description

-   Adds the ability to assign different prices to different SOPs

# Testing instructions

-   Complete a number of multi product journeys and check that you can define a price per SOP
-   Check the validation for the sop price functions
-   Check that the price per sop is included in the matching JSON

# Type of change
